assets/scss/main.scss

```diff
@@ -35,6 +35,21 @@ $component-toolbar: (
 	name: 'md-toolbar-tools',
 );
 
+$component-whiteframe: (
+    theme: map-get($config, theme),
+    name: 'md-whiteframe-z1'
+);
+
+$component-tileFooter: (
```

Comment:
Name of variable `component-tileFooter` should be written in all lowercase letters with hyphens instead of underscores